packages/flutter_tools/test/general.shard/base/logger_test.dart

```diff
@@ -30,8 +30,14 @@ void main() {
       fakeStopWatch = FakeStopwatch();
     });
     testUsingContext('error', () async {
-      final BufferLogger mockLogger = BufferLogger();
-      final VerboseLogger verboseLogger = VerboseLogger(mockLogger);
+      final BufferLogger mockLogger = BufferLogger(
+        terminal: AnsiTerminal(
+          stdio: stdio, // Danger, using real stdio
```

Comment:
Maybe also explain the reason for using the real stdio.

Reply:
Fixed by switching to mock stdio